### PR TITLE
OPS-5899 Adopt old module for cloudflare firewall rules

### PR DESCRIPTION
**Pull request by snovikov**
Cloudflare resources are scheduled for deprecation on 1st of May 2024:
- [cloudflare_filter](https://registry.terraform.io/providers/cloudflare/cloudflare/4.29.0/docs/resources/filter)
- [cloudflare_firewall_rule](https://registry.terraform.io/providers/cloudflare/cloudflare/4.29.0/docs/resources/firewall_rule)

Instead of them a new resource is introduced (`ruleset`), which replace current functionality of abovementioned resources and extend them with additional phases of CDN request processing:
- [cloudflare_ruleset](https://registry.terraform.io/providers/cloudflare/cloudflare/4.29.0/docs/resources/ruleset)

We adopt current state of [terraform-cloudflare-firewall:v1.4.0](https://github.com/Flaconi/terraform-cloudflare-firewall/tree/v1.4.0) of firewall module to continue further module developement.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -235,7 +235,7 @@ _lint-fmt: _pull-tf
 	@echo "# *.tf files"
 	@echo "------------------------------------------------------------"
 	@if docker run $$(tty -s && echo "-it" || echo) --rm -v "$(CURRENT_DIR):/t:ro" --workdir "/t" hashicorp/terraform:$(TF_VERSION) \
-		fmt -recursive -check=true -diff=true -write=false -list=true .; then \
+		fmt -recursive -check=true -diff=true -write=true -list=true .; then \
```
Comment (cytopia) [inline Makefile:238]: FYI: All `lint` stages are currently read-only and do not change anything. Why the change here?

Comment (snovikov) [reply Makefile:238]: copy & paste from here https://github.com/Flaconi/terraform-cloudflare-zone/blob/553805b3a99856eea8f62815bb50651ad4ead3eb/Makefile#L238

**Review by cytopia — APPROVED** [review]
